cpp/include/cudf/detail/interop.hpp

```diff
@@ -74,6 +74,18 @@ std::shared_ptr<arrow::Array> to_arrow_array(cudf::type_id id, Ts&&... args)
     case type_id::TIMESTAMP_NANOSECONDS:
       return std::make_shared<arrow::TimestampArray>(arrow::timestamp(arrow::TimeUnit::NANO),
                                                      std::forward<Ts>(args)...);
+    case type_id::DURATION_SECONDS:
```

Comment:
@rgsl888prabhu I made the changes based on #5778, could you review when you get a chance?